vim-patch:8.2.{3905,3909}: Dockerfile, Containerfile using prefix name not recognized

clason opened the pull request:
Problem:    Dockerfile using prefix name not recognized.
Solution:   Recognize Dockerfile.*. (closes vim/vim#9410)
https://github.com/vim/vim/commit/c88ac94a0bdc70012558813404821fe077f1de20

clason committed:
vim-patch:8.2.3905: Dockerfile using prefix name not recognized

Problem:    Dockerfile using prefix name not recognized.
Solution:   Recognize Dockerfile.*. (closes vim/vim#9410)
https://github.com/vim/vim/commit/c88ac94a0bdc70012558813404821fe077f1de20

zeertzjq commented:
I guess <https://github.com/vim/vim/commit/bbdd3fb804bf69684f7b0f49d3c0a0b039daec02> can be ported together.

clason committed:
vim-patch:8.2.3909: Containerfile using prefix name not recognized

Problem:    Containerfile using prefix name not recognized.
Solution:   Recognize Containerfile.*.
https://github.com/vim/vim/commit/bbdd3fb804bf69684f7b0f49d3c0a0b039daec02

seandewar reviewed (commented):
Missing https://github.com/vim/vim/commit/c88ac94a0bdc70012558813404821fe077f1de20#diff-c07c0771a9f5af08e375703487a8fe7bbddd71f98a7bbe06acd78365525a243eR400 otherwise LGTM (probably because there's a typo that should've been fixed in the port of 8.2.0190
